ui/transactions_page.go

```diff
@@ -261,10 +262,9 @@ func (page *transactionsPage) Handle(common pageCommon) {
 
 func initTxnWidgets(common *pageCommon, transaction *wallet.Transaction, txWidgets *transactionWdg) {
 	txWidgets.amount = common.theme.Label(unit.Dp(16), transaction.Balance)
-	txWidgets.time = common.theme.Body1("Pending")
+	txWidgets.time = common.theme.Body1(dcrlibwallet.ExtractDateOrTime(transaction.Txn.Timestamp))
```

Comment:
you can have the "DateTime" added directly to the Transaction data type. That way, you can use it in the layout without calling the ExtractDateOrTime method.